playwright.config.ts

```diff
@@ -0,0 +1,77 @@
+import { defineConfig, devices } from "@playwright/test";
```

Comment:
I think this needs the copyright header.

Reply:
right, let me see why it was not caught up by CI